dspace/config/modules/solr-statistics.cfg

```diff
@@ -13,7 +13,7 @@
 solr-statistics.server = ${solr.server}/statistics
 
 # A comma-separated list that contains the bundles for which the bitstreams will be displayed
-solr-statistics.query.filter.bundles=ORIGINAL
+solr-statistics.query.filter.bundles=ORIGINAL, IIIF
```

Comment:
Do we have an easy way to add a IIIF bundle? Or is that just expected to occur by editing an Item and uploading to a bundle of that name?  Just trying to understand the process of using this bundle and whether it will be "connected" into the submission process eventually.

Reply:
The IIIF bundle needs to be removed from this configuration.   

The 4Science approach no longer uses a designated bundle for IIIF. I think the current submission process and UI will accommodate the typical case quite well, especially after adding of a few new metadata fields for bitstreams.  That addition hasn't been proposed yet but it might be something to add in 7.1.

In the new approach there can (optionally) be multiple bundles and bundles can carry IIIF-related metadata. @abollini knows more about this since it is part of the 4Science implementation.